components/net/http_loader.rs

```diff
@@ -1066,11 +1107,13 @@ pub fn load<A, B>(load_data: &LoadData,
         // Send an HttpResponse message to devtools with the corresponding request_id
         // TODO: Send this message even when the load fails?
         if let Some(pipeline_id) = load_data.pipeline_id {
+            if devtools_chan.is_some() {
```

Comment:
`let Some` then take reference directly would be tidier? Not sure the motivation of using `as_ref` here ;)